dart_challenge/bin/dart_challenge.dart

```diff
@@ -30,6 +31,9 @@ void main(List<String> arguments) {
       case 1:
         dart_challenge.exercise1();
         break;
+      case 2:
+        print(unit_test_1.unitTest1(31));
```

Comment:
- Cho người dùng nhập từ bán phím số cần tìm ước
- Trong print cho thêm text chú thích ví dụ như "Divisor of ...."